Index: gettext-0.17/gettext-tools/src/read-catalog-abstract.c
===================================================================
--- gettext-0.17.orig/gettext-tools/src/read-catalog-abstract.c
+++ gettext-0.17/gettext-tools/src/read-catalog-abstract.c
@@ -169,6 +169,8 @@ catalog_reader_parse (abstract_catalog_r
 		      const char *real_filename, const char *logical_filename,
 		      catalog_input_format_ty input_syntax)
 {
+  error_message_count = 0;
+
   /* Parse the stream's content.  */
   parse_start (pop);
   input_syntax->parse (pop, fp, real_filename, logical_filename);
@@ -181,7 +183,6 @@ catalog_reader_parse (abstract_catalog_r
 				    "found %d fatal errors",
 				    error_message_count),
 			  error_message_count));
-  error_message_count = 0;
 }
 
 
